### DISCUSSION TITLE
android Error: route ip+net: netlinkrib: permission denied

### DISCUSSION THREAD

**Discussion by pokegamer5547** [claudiodangelis/qr-filetransfer#321] [Q&A] [answered]
I'm not so techy enough to understand but, i used qrcp for long time on android via termux without any issues. But with android 13 update, it starts saying
`Error: route ip+net: netlinkrib: permission denied`
I tried using "any" for interface --bind and got this error.
![Screenshot_20231221-224436_Termux](https://github.com/claudiodangelis/qrcp/assets/65234485/e4b0d3eb-7933-4d15-8cf9-f5d257796c6d)

After looking around, i found out it's the golang [issue](https://github.com/golang/go/issues/40569)
I tried using a fix mentioned in someplaces where it involves patching go to get ip list only. 
https://github.com/wlynxg/anet
Butfor even after that qrcp still isn't working.

**Comment by claudiodangelis** [comment]
Hello, thanks for pointing this out.
I will try to reproduce the issue locally and see if we can find a solution!
Claudio

**Reply by pokegamer5547** [reply, marked as answer]
After digging more,  i decided to modify my config file too and change it there to "any" as interface and "192.0.0.2" for bind and it's working again now! 
It just doesn't work when i want it to only listen on single specific interface.

**Reply by claudiodangelis** [reply]
Thanks a lot for this!! I will add a note to the README as other users might be affected!
